src/components/TrancheInvestment.tsx

```diff
@@ -84,18 +84,21 @@ export const TrancheInvestment: FunctionComponent<TrancheInvestmentProps> = ({
 				userTrancheBalance,
 				deal.interestFee
 			);
-			setProjectedReturns(projectedReturns);
+			const roundedProjectedReturns = round(projectedReturns, Big.roundDown, 0);
+			setProjectedReturns(roundedProjectedReturns);
 
 			const currentReturns = investorCurrentReturns(tranche, userTrancheBalance);
-			setCurrentReturns(currentReturns);
+			const roundedCurrentReturns = round(currentReturns, Big.roundDown, 0);
+			setCurrentReturns(roundedCurrentReturns);
 		}
 	}, [tranche, deal, userTrancheBalance]);
 
 	useEffect(() => {
 		if (userTrancheBalance && currentReturns && amountWithdrawn) {
 			const currentValue =
 				userTrancheBalance.uiAmount + currentReturns.toNumber() - amountWithdrawn.uiAmount;
-			setCurrentValue(currentValue);
+			const roundedCurrentValue = round(currentValue, Big.roundUp, 0);
```

Comment:
again, rounding is ui logic and I'd like to keep it as close to the ui as possible. rounding should happen at the last moment not during general component logic imo.